bulk_adding/templates/bulk_add/add_form.html

```diff
@@ -57,10 +57,15 @@
   {% endfor %}
 </ul>
 <script type="text/html" class="js-bulk-known-person-alternate-name-form">
-    <form action="." method="post">
+<form action="" method="post">
+        <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
         <p>
             <label for="alt-name">Name as it appears on nomination paper:</label>
-            <input id="alt-name" type="text">
+            <input id="alt-name" name="name" type="text">
+        </p>
+        <p>
```

Comment:
Since the button explicitly says it's an alternative name from the nomination paper, I think this can be a hidden field.